code/modules/projectiles/gun_system.dm

```diff
@@ -938,11 +938,11 @@ and you're good to go.
 
 	if(dual_wield) //akimbo firing gives terrible accuracy
 		if(gun_skill_category == GUN_SKILL_PISTOLS)
-			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.1*rand(1,2))
-			gun_scatter += 10*rand(1,3)
+			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.5*rand(1,2))
```

Comment:
accuracy is a wack mechanic and shoulndt be mostly used for anything, most stuff should miss if it hits, use scatter
```suggestion
			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.1*rand(1,2))
```

Reply:
Decrceased the accuracy nerf heavily in the latest commit.
It still has 10 max, but that's quite negligible.